src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String REGEX_CONST = ",";
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int buy = 0;
+        int supply = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] stringArray = line.split(REGEX_CONST);
+                if (stringArray[0].equals("supply")) {
+                    supply += Integer.parseInt(stringArray[1]);
+                } else if (stringArray[0].equals("buy")) {
+                    buy += Integer.parseInt(stringArray[1]);
```

Comment:
let's declare 0, 1 supply and buy as constants, that will improve code readability